01_Data/src/layers/sequelize/model/TransactionEvent/Transaction.ts

```diff
@@ -41,17 +41,17 @@ export class Transaction extends Model implements OCPP2_0_1.TransactionType {
   declare isActive: boolean;
 
   @HasMany(() => TransactionEvent, { as: Transaction.TRANSACTION_EVENTS_ALIAS, foreignKey: 'transactionDatabaseId' })
-  declare transactionEvents?: OCPP2_0_1.TransactionEventRequest[];
```

Comment:
`@HasMany` here defines an association between `Transaction` and `TransactionEvent` tables. It would better to use `TransactionEvent[]` as well as at line 48 and 78.